components/checkbox/Checkbox.js

```diff
@@ -22,6 +22,7 @@ const factory = (Check) => {
       onMouseEnter: PropTypes.func,
       onMouseLeave: PropTypes.func,
       style: PropTypes.object,
+      testid: PropTypes.string,
```

Comment:
Is there no other way to do it? We want to remove all forks, unnecessary overhead of maintaining these repos.